__tests__/SetsSelectors.test.js

```diff
@@ -1504,6 +1524,13 @@ describe('SetsSelectors', () => {
                 expect(result).toBe(null);
             });
 
+            test('return null if set is deleted', () => {
+
+                const result = sut.getFastestAvgVelocityEver(state, deletedSet);
+
+                expect(result).toBe(null);
+            });
+
```

Comment:
- [ ] Test where the set you're checking for ISN'T deleted but there are deleted sets that have the same exercise name / tags / etc.